Errors when running npm

0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'serve' ]
2 info using npm@2.15.5
3 info using node@v4.4.5
4 verbose run-script [ 'preserve', 'serve', 'postserve' ]
5 info preserve bare-redux@1.0.0
6 info serve bare-redux@1.0.0
7 verbose unsafe-perm in lifecycle true
8 info bare-redux@1.0.0 Failed to exec serve script
9 verbose stack Error: bare-redux@1.0.0 serve: `npm run copy && npm run serve:watch`
9 verbose stack Exit status 1
9 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\lifecycle.js:217:16)
9 verbose stack     at emitTwo (events.js:87:13)
9 verbose stack     at EventEmitter.emit (events.js:172:7)
9 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\spawn.js:24:14)
9 verbose stack     at emitTwo (events.js:87:13)
9 verbose stack     at ChildProcess.emit (events.js:172:7)
9 verbose stack     at maybeClose (internal/child_process.js:827:16)
9 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
10 verbose pkgid bare-redux@1.0.0
11 verbose cwd H:\just-react
12 error Windows_NT 10.0.10240
13 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "serve"
14 error node v4.4.5
15 error npm  v2.15.5
16 error code ELIFECYCLE
17 error bare-redux@1.0.0 serve: `npm run copy && npm run serve:watch`
17 error Exit status 1
18 error Failed at the bare-redux@1.0.0 serve script 'npm run copy && npm run serve:watch'.
18 error This is most likely a problem with the bare-redux package,
18 error not with npm itself.
18 error Tell the author that this fails on your system:
18 error     npm run copy && npm run serve:watch
18 error You can get information on how to open an issue for this project with:
18 error     npm bugs bare-redux
18 error Or if that isn't available, you can get their info via:
18 error
18 error     npm owner ls bare-redux
18 error There is likely additional logging output above.
19 verbose exit [ 1, true ]


``From my cmd is below:``

Your environment has been set up for using Node.js 4.4.5 (x64) and npm.

H:\just-react>npm install
npm WARN peerDependencies The peer dependency babel-core@^6.0.0 included from babel-loader will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN optional dep failed, continuing fsevents@1.0.12
npm WARN optional dep failed, continuing fsevents@1.0.12
npm WARN optional dep failed, continuing fsevents@1.0.12
npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.5

npm ERR! shasum check failed for C:\Users\SEVERE~1\AppData\Local\Temp\npm-3840-2b6667c0\registry.npmjs.org\weinre\-\weinre-2.0.0-pre-I0Z7U9OV.tgz
npm ERR! Expected: fef8aa223921f7b40bbbbd4c3ed4302f6fd0a813
npm ERR! Actual:   69b851383fd1bb4145db2c98492ebd579f8428b5
npm ERR! From:     https://registry.npmjs.org/weinre/-/weinre-2.0.0-pre-I0Z7U9OV.tgz
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     H:\just-react\npm-debug.log

H:\just-react>npm run serve

> bare-redux@1.0.0 serve H:\just-react
> npm run copy && npm run serve:watch


> bare-redux@1.0.0 copy H:\just-react
> cp src/index.html dest/index.html

'cp' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "copy"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.5
npm ERR! code ELIFECYCLE
npm ERR! bare-redux@1.0.0 copy: `cp src/index.html dest/index.html`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bare-redux@1.0.0 copy script 'cp src/index.html dest/index.html'.
npm ERR! This is most likely a problem with the bare-redux package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cp src/index.html dest/index.html
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bare-redux
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls bare-redux
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     H:\just-react\npm-debug.log

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "serve"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.5
npm ERR! code ELIFECYCLE
npm ERR! bare-redux@1.0.0 serve: `npm run copy && npm run serve:watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bare-redux@1.0.0 serve script 'npm run copy && npm run serve:watch'.
npm ERR! This is most likely a problem with the bare-redux package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run copy && npm run serve:watch
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bare-redux
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls bare-redux
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     H:\just-react\npm-debug.log

H:\just-react>npm install babel-preset-stage-1 --save-dev
babel-preset-stage-1@6.5.0 node_modules\babel-preset-stage-1
├── babel-plugin-transform-class-properties@6.9.1 (babel-plugin-syntax-class-properties@6.8.0, babel-runtime@6.9.2)
├── babel-plugin-transform-export-extensions@6.8.0 (babel-plugin-syntax-export-extensions@6.8.0, babel-runtime@6.9.2)
├── babel-plugin-transform-decorators@6.8.0 (babel-plugin-syntax-decorators@6.8.0, babel-helper-define-map@6.9.0, babel-template@6.9.0, babel-types@6.9.1, babel-helper-explode-class@6.8.0, babel-runtime@6.9.2)
├── babel-plugin-transform-class-constructor-call@6.8.0 (babel-plugin-syntax-class-constructor-call@6.8.0, babel-template@6.9.0, babel-runtime@6.9.2)
└── babel-preset-stage-2@6.5.0 (babel-plugin-syntax-trailing-function-commas@6.8.0, babel-plugin-transform-object-rest-spread@6.8.0, babel-preset-stage-3@6.5.0)

H:\just-react>npm install
npm WARN optional dep failed, continuing fsevents@1.0.12
lite-server@2.2.0 node_modules\lite-server
├── connect-history-api-fallback@1.2.0
├── minimist@1.2.0
├── connect-logger@0.0.1 (moment@2.13.0)
├── lodash@4.13.1
└── browser-sync@2.12.10 (emitter-steward@1.0.0, dev-ip@1.0.1, qs@6.2.0, immutable@3.8.1, ua-parser-js@0.7.10, opn@3.0.3, browser-sync-client@2.4.2, http-proxy@1.13.3, portscanner@1.0.0, serve-static@1.10.2, connect@3.4.1, resp-modifier@6.0.1, chokidar@1.5.1, micromatch@2.3.7, fs-extra@0.26.7, serve-index@1.7.3, socket.io@1.4.6, yargs@4.7.1, bs-recipes@1.2.2, localtunnel@1.8.1, easy-extender@2.3.2, eazy-logger@2.1.3, browser-sync-ui@0.5.19)

H:\just-react>npm run serve

> bare-redux@1.0.0 serve H:\just-react
> npm run copy && npm run serve:watch


> bare-redux@1.0.0 copy H:\just-react
> cp src/index.html dest/index.html

'cp' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "copy"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.5
npm ERR! code ELIFECYCLE
npm ERR! bare-redux@1.0.0 copy: `cp src/index.html dest/index.html`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bare-redux@1.0.0 copy script 'cp src/index.html dest/index.html'.
npm ERR! This is most likely a problem with the bare-redux package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cp src/index.html dest/index.html
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bare-redux
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls bare-redux
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     H:\just-react\npm-debug.log

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "serve"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.5
npm ERR! code ELIFECYCLE
npm ERR! bare-redux@1.0.0 serve: `npm run copy && npm run serve:watch`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bare-redux@1.0.0 serve script 'npm run copy && npm run serve:watch'.
npm ERR! This is most likely a problem with the bare-redux package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run copy && npm run serve:watch
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs bare-redux
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls bare-redux
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     H:\just-react\npm-debug.log

H:\just-react>

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "scripts": {
     "serve": "npm run copy && npm run serve:watch",
     "serve:watch": "concurrently --kill-others \"npm run webpack:watch\" \"npm run lite-server\"",
-    "copy": "cp src/index.html dest/index.html",
+    "copy": "copyfiles -f src/*.html dest",
     "lite-server": "lite-server",
     "webpack:watch": "webpack --watch"
   },
@@ -27,6 +27,7 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.5.0",
     "concurrently": "^2.1.0",
+    "copyfiles": "^0.2.1",
     "lite-server": "^2.2.0",
     "webpack": "^1.13.1"
   }

